cmd/convox/ps.go

```diff
@@ -44,6 +49,9 @@ func init() {
 }
 
 func cmdPs(c *cli.Context) error {
+	stdcli.NeedHelp(c)
+	stdcli.NeedArg(c, 0)
```

Comment:
Pushed an update to only display the error if `DEBUG` or `CONVOX_DEBUG` are non-empty. Would that work?